test/pytest/service-desktop/test_security.py

```diff
@@ -1,33 +1,73 @@
 # Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
 # For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+import logging
 
 import pytest
 import time
 
 from harness.interface.defs import status
-
+from harness.api.developermode import PhoneModeLock
+from harness.api.security import GetPhoneLockStatus, GetPhoneLockTime, SetPhoneLockOff
+from harness.request import TransactionError
+
+
+class SecurityTester:
+    def __init__(self, harness):
+        self.harness = harness
+
+    def assert_phone_unlocked(self):
+        assert GetPhoneLockStatus().run(self.harness).response.status == status["NoContent"]
+
+    def assert_phone_locked(self):
+        try:
+            GetPhoneLockStatus().run(self.harness).response.status
```

Comment:
there are general fixtures for QAMuditaOS pytest- please change the usage there too.

Reply:
Do you mean fixtures from conftest.py, like _@pytest.mark.usefixtures("phone_locked")_?
They are using developer mode EP API, not security EP API -> no changes are needed